src/js/actions/layers.js

```diff
@@ -881,11 +881,8 @@ define(function (require, exports) {
             return Promise.resolve();
         }
 
-        if (modifier !== undefined && layerSpec.size > 1) {
-            return Promise.reject(new Error("Select with modifier requires a single layer"));
-        }
-
-        var selected = document.layers.selected,
+        var setPivot = modifier === undefined || layerSpec.size === 1,
```

Comment:
No modifier just means "make the current selection equal to the set of layers passed in". If you provide a modifier, we have to do some calculation that may take a pivot into account. That's why no modifier means all is well.